vignettes/nbbp.Rmd

```diff
@@ -0,0 +1,315 @@
+---
+title: "nbbp"
+output:
+  rmarkdown::html_vignette:
+    fig_width: 7
+    fig_height: 5
+vignette: >
+  %\VignetteIndexEntry{nbbp}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo = F}
+library(knitr)
+knitr::opts_chunk$set(
+  out.extra = 'style="display:block; margin:auto;"'
+)
+```
+
+Welcome to `nbbp`, an R package for inference of parameters of negative binomial branching process models.
+This vignette covers the basics of inferring parameters from simple data, where every chain size is known exactly and our data includes every possible size.
```

Comment:
"our data includes every possible size" sounds like you have actually observed every possible size, which of course may not be true. I would revise the larger clause to "where every possible chain size is observable, and every observed chain size is known exactly" or something like that.